jgrapht-core/src/main/java/org/jgrapht/alg/VertexCovers.java

```diff
@@ -147,6 +147,60 @@
 
         return cover;
     }
+    
+    /**
+     * Finds an optimal vertex cover of a specified graph. At each iteration,
+     * the algorithm picks a random vertex v and distinguishes two cases: 
+     * either v has to be added to the vertex cover or all neighbors of v.
+     * 
+     * Let k be the size of an optimal vertex cover. Then the algorithm requires
+     * O(2^k * poly(n)) time to terminate. Thus, this algorithm achieves polynomial
+     * running time on graph classes where k = O(log^c n) for any constant c.
+     *
+     * @param g the graph for which vertex cover approximation is to be found.
+     *
+     * @return a set of vertices which is an optimal vertex cover for the specified
+     * graph.
+     */
+    public static <V, E> Set<V> findVertexCover(Graph<V, E> g)
```

Comment:
findVertexCover -> findMinimumVertexCover
To emphasize that you are not just looking for a general vertex cover, but a minimum one.